### PR TITLE
Refactor/cleanup gateway

### PR DESCRIPTION
**Pull request by simonhir**
**Description**

Cleanup gateway:
- rm unneeded no security properties file
- cleanup properties to match integrations
- add local properties

### CODE REVIEW

File: refarch-gateway/src/main/resources/application.yml [hunk 1]
```diff
@@ -75,5 +75,14 @@ info:
...
 
+refarch:
+  security:
+    sso-issuer-url:
+    sso-client:
+    sso-secret:
+    cors:
+      allowed-origins-public:
+    csrf-whitelisted:
+
```
Comment (simonhir) [inline refarch-gateway/src/main/resources/application.yml:86]: @devtobi what do you think should i remove that or let it?

**Review by devtobi — DISMISSED** [review]
LGTM

File: refarch-gateway/src/main/resources/application.yml (hunk 1 above)
Comment (simonhir) [reply refarch-gateway/src/main/resources/application.yml]: Reverted

**Review by devtobi — APPROVED** [review]
LGTM

**Pull request merged**